libs/ui-shared/src/lib/components/atoms/dropdown/Dropdown.js

```diff
@@ -0,0 +1,3 @@
+import { Dropdown } from "antd";
+
```

Comment:
suggest using dropdown component created in sp-student (which needs addition of proptypes) to avoid refactoring of molecules/organisms using that compnent.

Reply:
I already refactored it. So the default dropdown from AntDesign will work for us.